### PR TITLE
Jepsen 0.3.01 upgrade

### PR DESCRIPTION
**Pull request by qvad**

### CODE REVIEW

**Review by frozenspider — APPROVED** [review]

File: yugabyte/src/yugabyte/workaround.clj [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+(defn process-index
+  "Maps processes to columns"
+  [history]
+  (->> history
+       history/processes
+       history/sort-processes
+       (reduce (fn [m p] (assoc m p (count m)))
+               {})))
+
+(defn html
```
Comment (frozenspider) [inline yugabyte/src/yugabyte/workaround.clj]: It seems that the current implementation of `timeline/html` is not very different to this, why can't we use it?

File: yugabyte/src/yugabyte/bank_improved.clj [hunk 2]
```diff
@@ -121,40 +119,37 @@
...
+                                (merge {:count (count errs)
+                                        :first (first errs)
+                                        :worst (util/max-by
+                                                 (partial bank/err-badness test)
+                                                 errs)
+                                        :last  (peek errs)}
+                                       (if (= type :wrong-total)
+                                         {:lowest  (util/min-by :total errs)
+                                          :highest (util/max-by :total errs)}
+                                         {}))]))
```
Comment (frozenspider) [inline yugabyte/src/yugabyte/bank_improved.clj:152]: Looks like only whitespaces were changed here?

File: yugabyte/src/yugabyte/bank_improved.clj [hunk 3]
```diff
@@ -13,12 +13,10 @@
...
   (:refer-clojure :exclude
                   [test])
-  (:require [clojure [pprint :refer [pprint]]]
-            [clojure.tools.logging :refer [info]]
-            [jepsen.tests.bank :as bank]
-            [knossos.op :as op]
+  (:require [jepsen.tests.bank :as bank]
             [clojure.core.reducers :as r]
             [jepsen
+             [history :as h]
```
Comment (frozenspider) [inline yugabyte/src/yugabyte/bank_improved.clj:19]: Interesting, I don't see `history.clj` in `jepsen` - but I see it used all over Jepsen

Comment (qvad) [reply yugabyte/src/yugabyte/bank_improved.clj:19]: It's separated package now https://github.com/jepsen-io/history

File: yugabyte/src/yugabyte/workaround.clj (hunk 1 above)
Comment (qvad) [reply yugabyte/src/yugabyte/workaround.clj]: Trying to understand what's needs to be changed here https://github.com/jepsen-io/jepsen/issues/564
These  are the only tests (single-multi-key-acid) where checker is failed

**Pull request merged**